src/resources/views/crud/fields/upload_multiple.blade.php

```diff
@@ -120,6 +120,16 @@
 			font-weight: 400!important;
 		}
 
+		.backstrap-file-label[has-selected-files=true] {
+			display: inline-table;
+			width: 100%;
+		}
```

Comment:
One problem... when we have many files selected, the "browse" button no longer looks ok, as if it is part of the input:
![CleanShot 2024-07-24 at 17 51 59@2x](https://github.com/user-attachments/assets/49dc8f57-3d31-42e9-8dd1-1737dbf45e28)

![CleanShot 2024-07-24 at 17 56 38@2x](https://github.com/user-attachments/assets/118500ac-0e87-451f-9b55-b7658e7a349f)

Don't know if we should struggle to fix this with CSS... or we should put the files in a span above the browse field. After all... when they do get uploaded, it will still end up looking like that (with files above the input):
![CleanShot 2024-07-24 at 18 00 06@2x](https://github.com/user-attachments/assets/4a41c2d1-8ce6-4c50-977a-b4f0265a7245)

Reply:
Indeed, the "browse" button is not the "best looking", maybe try to vertically center it ? I really hate that feature of putting each file in a row after upload.  
In your example the `[X]` more or less is aligned because you are using hashes for names, if using "slugs" as most of the times we use, it looks ugly X10. Plus, that whole space empty .. it could easily be just 2 lines instead of 5. 

I tried the "pills" above the field, they look good too, but we always have the input space "empty" and I thought to use it. I think "pills" look way more cleaner. Will do the changes and ping you to have a second look.